chsdi/models/vector/stopo.py

```diff
@@ -288,10 +299,36 @@ class LetztEisMaxMetadata (Base, ShopProductClass, Vector):
     __table_args__ = ({'schema': 'geol', 'autoload': False})
     __tablename__ = 'view_gridstand_gklgm500'
     __bodId__ = 'ch.swisstopo.gklgm500-papierkarte.metadata'
+    price = Column('price', Integer)
 
 register('ch.swisstopo.gklgm500-papierkarte.metadata', LetztEisMaxMetadata)
 
 
+class GridstandPk500(Base, ShopProductClass, Vector):
+    __tablename__ = 'view_gridstand_datenhaltung_pk500_tilecache_shop'
+    __table_args__ = ({'schema': 'datenstand', 'autoload': False})
+    __bodId__ = 'ch.swisstopo.pixelkarte-farbe-pk500.noscale'
+    price = Column('price', Integer)
+
+register('ch.swisstopo.pixelkarte-farbe-pk500.noscale', GridstandPk500)
+
+
+class Icao500Digital (Base, ShopProductClass, Vector):
+    __table_args__ = ({'schema': 'datenstand', 'autoload': False})
+    __tablename__ = 'view_gridstand_icao500_dig'
+    __bodId__ = 'ch.bazl.luftfahrtkarten-icao'
```

Comment:
Meaning that for next deploy we'll not have the price for Fertigsprodukte?
If this is the case, then for me we should not go in prod with these layers.
@mariokeusen @gjn 
I will crosscheck with Markus Heilig next monday